assembly/src/assembler/instruction/field_ops.rs

```diff
@@ -3,56 +3,62 @@ use super::{
     StarkField, ONE, ZERO,
 };
 use crate::MAX_EXP_BITS;
+use vm_core::TWO;
```

Comment:
minor nit: I would probably not introduce constant `TWO` in `core` but rather declare it here. So far, we've put only `ONE` and `ZERO` in core as putting other values there will probably result in quite a few constant declarations.

Reply:
Done. Thanks.